src/sectionproperties/post/fibre.py

```diff
@@ -0,0 +1,198 @@
+"""Provides functionalities to export a section to a fibre section.
+
+It can be used in [suanPan](https://github.com/TLCFEM/suanPan) to perform further FEA.
+"""
+from __future__ import annotations
+
+from abc import abstractmethod
+
+from numpy import ndarray
+
+from sectionproperties.analysis import Section, solver
+from sectionproperties.analysis.fea import Tri6, shape_function
+from sectionproperties.pre import CompoundGeometry, Geometry
+
+
+class Cell:
+    """Holds the information of a fibre cell."""
+
+    tag: int
+    area: float
+    material: str
+    y: float
+    z: float
+    omega: float
+    py: float
+    pz: float
+
+    def __init__(self, ele: Tri6, omega: ndarray = None):
+        """Converts a Tri6 element to a fibre cell.
+
+        If `omega` is None, no warping is considered.
+
+        Args:
+            ele (Tri6): The Tri6 element
```

Comment:
Don't need the types next to the argument name - sphinx napoleon handles this for us. 

e.g. 
```
Args:
   ele: The Tri6 element
```

General comment for all docstrings.